concourse/scripts/generate-cf-secrets.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+set -x
+
+SCRIPTS_DIR=`dirname $0`
+OUTPUT_FILE=$1
+
+pwgen() {
+   openssl rand -base64 12 | sed 's/+/0/;s/\//0/'
+}
+cat > $OUTPUT_FILE <<EOF
+---
+  secrets:
+    vcap_password: $(pwgen)
```

Comment:
This needs to be generated in a different way, it must be a sha-512 of the password. For instance  `pwgen | mkpasswd -m sha-512 -s` 

But it would be convenient if you store also the cleartext password in this file, to be used for troubleshooting...